px-central/README.md

```diff
@@ -0,0 +1,142 @@
+# PX-Central
```

Comment:
Can we call this PX-Backup... since it shows up as PX-Backup as a product in AWS marketplace ?

Reply:
If for the marketplace, if px-backup needs to be enabled by default, we have to change the values.yaml to make it enabled by default. But renaming the chart name to px-backup, we will then break the uniformity across. I think in all docs related to market place we can mention the product as px-backup but internallyt the chart can remain px-central.

Reply:
I was in dilemma on the naming convention due to maintaining it uniform with the existing product. Are you guys ok with keeping the name as px-central?

Reply:
@diptiranjanpx I enabled `px-backup` to be installed by default and pushed the changes.